src/cnc/models/providers/amazon/environment_collection.py

```diff
@@ -16,6 +16,12 @@ class AWSEnvironmentCollection(EnvironmentCollection):
     # Properties
     # ------------------------------
 
+    @property
+    def has_service_domains(self):
+        if self.application.flavor == "lambda-lite":
```

Comment:
Similarly, I think we also need to early return here since we're not using a routing lambda for this flavor at the moment: https://github.com/coherenceplatform/cnc/pull/90/files#diff-6f4d89686bbbec86790058a42a966c92575a5cff10e9054b090aae9ea8226ea1R174

Reply:
What do you mean, could you explain a little better, I didn't understand, sorry

Reply:
Ah sorry that link seems like it doesn't take you where it's supposed to, in this file there's this generate tf assets function, it should be updated with this same early return logic like this:
```python
    def generate_tf_assets(self, config_files_path, rendered_files_path):
        if self.application.flavor == "lambda-lite":
            return True

        log.debug(f"Generating provider assets for {self}...")
        # Zip the js file
        # ensure its in env_collection config_files_path
        zip_file_path = os.path.join(rendered_files_path, "frontend_routing_lambda")
        shutil.make_archive(
            zip_file_path,
            "zip",
            root_dir=(f"{config_files_path}/frontend_routing_lambda"),
        )

        return True
```